### ISSUE TITLE
[Bug] Cannot Download as PDF

### ISSUE THREAD

**Issue by erdemkosk** [AmruthPillai/Reactive-Resume#2108]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Cloud (https://rxresu.me)

### Current Behavior

Hi, after when i create resume. When ı try to download it navigate about:blank.
For this reason we cannot download resume.

### Expected Behavior

We can download as pdf

### Steps To Reproduce

Click download as pdf button and wait

### What browsers are you seeing the problem on?

Chrome

### What template are you using?

None

**Comment by robiningelbrecht** [comment]
Got the exact same issue

**Comment by ATHman3** [comment]
@erdemkosk, did you use the download button in the export section of the right sidebar? I’m not sure why, but it was blocked by Chrome’s pop-up blocker.

**Comment by nate235** [comment]
This is an old issues from 4-[5 months ago](https://github.com/AmruthPillai/Reactive-Resume/issues/1979). None of there work arounds mentioned have worked for me. Probably time to look for other apps to accomplish this unfortunately as this was a decent one while it worked.

**Comment by erdemkosk** [comment]
I guess this is related turkish characters. When renaming with english it solves. I guess this is the bug related that

**Comment by robiningelbrecht** [comment]
No it isn't. My resume is in English and I have the same issue

**Comment by jasper-clarke** [comment]
I am having this issue with the resume name "CV / Resume", all English characters.

**Comment by foglar** [comment]
I noticed that this problem occurs to me, only when I set font style to **Space Grotesk** font. Otherwise using other font (for example **Montserrat**) resolved issue in my case.

**Comment by luascfl** [comment]
https://en.wikipedia.org/wiki/Acute_accent

My error is solved by renaming the resume without a special character like: á à

**Comment by davehornik** [comment]
> Renaming the file name to English might work

**This one worked for me**. I had Czech latin letters in it.

**Comment by carsaig** [comment]
Same issue here. It's an english name. I can pick the pdf from my file structure but it won't render in the browser. The pop-up implication was there, I set the chrome config to allow pop-ups from this page - still no success. Super annoying when HR calls and tells me the download link was broken.

**Comment by simibac** [comment]
I had the same issue. I tried Chrome and realized that the application needs the permission to create a pop-up window, which might be disabled on your browser by default. After allowing it in the Safari Settings, it worked:

<img width="852" alt="Image" src="https://github.com/user-attachments/assets/3a38bd21-5c7b-43d7-af3a-eee6017940f1" />

**Comment by johnconnor-sec** [comment]
Tried all of the above and still get this erro rwhen trying to download PDF

![Image](https://github.com/user-attachments/assets/3b25bcdc-3c68-4d27-9978-5b2d867e9b8c)

**Comment by carsaig** [comment]
> I had the same issue. I tried Chrome and realized that the application needs the permission to create a pop-up window, which might be disabled on your browser by default. After allowing it in the Safari Settings, it worked:
> 
> <img alt="Image" width="852" src="https://private-user-images.githubusercontent.com/15369592/408893687-3a38bd21-5c7b-43d7-af3a-eee6017940f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzg3MTMzNDEsIm5iZiI6MTczODcxMzA0MSwicGF0aCI6Ii8xNTM2OTU5Mi80MDg4OTM2ODctM2EzOGJkMjEtNWM3Yi00M2Q3LWFmM2EtZWVlNjAxNzk0MGYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjA0VDIzNTA0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExODZiZTE2NjM4MTFlY2M3NmJkZmMwMjMyOGQwODM5NzgxYTU1OGE0Y2MxOWE3ZTVkYjEyNWQ3MjE2ZmYxNmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GpXm0ij0hrsqM8vyFB6sffPV9pVFagADDBuI-mZX3-A">

Unfortunately, this is not the solution. There is some implication with this feature...when attempting the download the first time, the browser flips the setting for a split second, then it just vanishes and runs into the error. Correcting the setting afterwards doesn't change a thing. No idea why.

**Comment by AMDphreak** [comment]
Is this still happening?
I previously had this same problem in Vivaldi, but not Chrome (which is what it is based off of), and it magically got fixed somehow.

edit: this is a duplicate of #2001

**Comment by prmbittencourt** [comment]
I am having the same issue after spending hours working on my resume. None of the solutions mentioned here have worked for me. The resume name has no special/accented/non-Latin characters (it's literally "basic") and the workaround suggested by @johnconnor-sec also does not work, as opening the link in my browser leads to a page where only the top 10 or 15% of the resume is rendered (it currently fits in one page, though there is a tiny bit of whitespace after the dotted line in the editor). Tested with Chromium and Firefox.

EDIT: I solved the issue by changing the version of the Chromium image in the docker-compose.yaml file from `2.18.0` to `latest`. There was a warning not to do that, so what sort of eldritch horrors can I expect to come drag me under now?

**Comment by prmbittencourt** [comment]
I made some changes and tried to export again and was faced with the same problem once more. Below is a screenshot of the share page.

<img width="1916" height="1080" alt="Image" src="https://github.com/user-attachments/assets/cd11fef8-404c-4115-bff7-ba6da71d8659" />

Then I restarted the container. The share page is still broken as above, but I could then export the resume. A couple of hours later, I tried exporting again and it failed. So it's something time-sensitive to how long ago the container was started?

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
